src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneSwitchConfig.java

```diff
@@ -0,0 +1,28 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.Accordion;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneSwitchConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Sends a notification when approaching the end of a lane. Settings below.")
```

Comment:
Remove the "settings below" part. this is the same look and feel as with every other skyhanni feature, no need to write this here.